www/app/app.ts

```diff
@@ -150,6 +150,7 @@ export class App {
     } else if (e instanceof errors.OperationTimedOut) {
       messageKey = 'error-timeout';
     } else if (e instanceof errors.ShadowsocksStartFailure) {
+      // TODO: only display this message if running on Windows.
```

Comment:
I think this applies to several error messages - let's add a TODO for `ConfigureSystemProxyFailure`, `NoAdminPermissions`, `UnsupportedRoutingTable`?

Reply:
Indeed, though `ShadowsocksStartFailure` is the only one used in other platforms.